plugins/cffi/cffitest.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```

Comment:
You can add the plugin in .travis.yml to get compiled tested and this check in tests/travis.sh to have a smoke test on each pr

Reply:
It looks like Travis is running on xenial which has a much too old pypy for this